src/megaapi_impl.cpp

```diff
@@ -7489,6 +7490,30 @@ bool MegaApiImpl::hasToForceUpload(const Node &node, const MegaTransferPrivate &
     return canForceUpload && (isMedia || isPdf) && !(hasPreview && hasThumbnail);
 }
 
+void MegaApiImpl::platformSetRLimitNumFile(int newNumFileLimit) const
+{
+#ifndef WIN32
+    struct rlimit rl{0,0};
+    if (0 < getrlimit(RLIMIT_NOFILE, &rl))
+    {
+        auto e = errno;
+        LOG_err << "Error calling getrlimit: " << e;
```

Comment:
We may want to return a boolean in this function, so it returns false in case of error. That way, the app knows its call has failed.

Reply:
If the OS won't accept the limit then I don't think there's really anything the app can do to improve its situation?  We already logged the issue.